src/components/articles/Articles.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ArticleBlock from "./ArticleBlock";
+import {Box, Grid} from "@mui/material";
+import {info} from "../../info/Info";
+
+export default function Articles() {
+    return (
+        <Box marginLeft="40px">
+            <Grid container display={'flex'} justifyContent={'center'} marginTop={'20px'}>
+                {info.articles.map((project, index) => (
+                   <Grid item xs={12} md={6} key={index}>
+                       <ArticleBlock image={project.image} source={project.source} title={project.title} />
+                   </Grid>
+                ))}
```

Comment:
**Suggestion:** Consider adding a unique key to each child in a list. This will help React identify which items have changed, are added, or are removed. Keys should be given to the elements inside the array to give the elements a stable identity.
```suggestion
                {info.articles.map((project) => (
                  <Grid item xs={12} md={6} key={project.source}>
                    <ArticleBlock image={project.image} source={project.source} title={project.title} />
                  </Grid>
                ))}
```